### PR TITLE
filelist file with updated grids

### PR DESCRIPTION
**Pull request by dalia2**

### CODE REVIEW

File: filelist.csv [hunk 1]
```diff
@@ -3,7 +3,7 @@ dvr90.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:4937,EPSG:5799,,dk_sdfe,A
 dnn.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:4937,EPSG:5733,,dk_sdfe,Agency for Data Supply and Efficiency,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
 fvr09.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:4937,EPSG:5317,,dk_sdfe,Agency for Data Supply and Efficiency,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
 nkgrf03vel_realigned_xy.ct2 + nkgrf03vel_realigned_z.gtx,DEFORMATION_MODEL,,,,,nordic_geodetic,The Nordic Geodetic Commission,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
-BWTA2017.gsb,HORIZONTAL_OFFSET,,EPSG:4314,EPSG:4258,,de_lgl_bw,www.lgl-bw.de,Data licence Germany - attribution - version 2.0 https://www.govdata.de/dl-de/by-2-0
+BWTA2017.gsb,HORIZONTAL_OFFSET,,EPSG:4314,EPSG:4258,,de_lgl,www.lgl-bw.de,Data licence Germany - attribution - version 2.0 https://www.govdata.de/dl-de/by-2-0
```
Comment (kbevers) [inline filelist.csv]: Hmmm... this probably shouldn't be changed?

Comment (dalia2) [reply filelist.csv]: I have edited the mistake

File: filelist.csv [hunk 2]
```diff
@@ -206,3 +206,9 @@ g2012bp0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:6641,,us_noa
 g2018u0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:5703,,us_noaa,NOAA,Public Domain
 g2018p0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:6641,,us_noaa,NOAA,Public Domain
 100800401.gsb,HORIZONTAL_OFFSET,,EPSG:4230,EPSG:4258,,es_cat_icgc,Institut Cartogràfic i Geològic de Catalunya (ICGC),Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+ISN93_ISN2016.gsb,HORIZONTAL_OFFSET,,EPSG:4659,EPSG:8086,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
```
Comment (rouault) [inline filelist.csv]: perhaps is_lmi as the source short name for consistency with others ?

File: filelist.csv [hunk 3]
```diff
@@ -206,3 +206,9 @@ g2012bp0.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:6319,EPSG:6641,,us_noa
...
+ISN2004_ISN2016.gsb,HORIZONTAL_OFFSET,,EPSG:5324,EPSG:8086,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN2004.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:5323,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN93.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:4945,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+Icegeoid_ISN2016.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,,EPSG:8085,EPSG:8089,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
+ISN_vel_beta.ct2 + ISN_vel_beta.gtx,VELOCITY_MODEL,,,,,lmi,National Land Survey of Iceland,Creative Commons Attribution 4.0 https://creativecommons.org/licenses/by/4.0/
```
Comment (rouault) [inline filelist.csv]: for completness could you add records like:
ISN93_ISN2016.gsb + ISN93_ISN2016.gtx,DEFORMATION_MODEL ... 
ISN2004_ISN2016.gsb + ISN2004_ISN2016.gtx,DEFORMATION_MODEL ...

**Comment by rouault** [comment]
Excellent !

**Comment by dalia2** [comment]
Thank you, for your help.

**Comment by rouault** [comment]
Now converted (for the horizontal and vertical grid shift only) to GeoTIFF in the next-gen proj-datumgrid-geotiff repo: https://github.com/OSGeo/proj-datumgrid-geotiff/tree/master/is_lmi

**Pull request merged**